github_visualization/github_stats.py

```diff
@@ -0,0 +1,289 @@
+import requests
+
+
+class GithubStatFetcher:
+    """A GitHub statistics fetcher for github repositories
+    """
+
+    def __init__(self, user_name, repositoty_name):
+        ''' Initialize GithubStatFetcher Object
+
+        Parameters
+        -------------
+        user_name : str
+           Name of the user or organization of the repository owner
+        repositoty_name : str
+           Name of repository
+        '''
+        self.user_name = user_name
+        self.repositoty_name = repositoty_name
+
+        self.baseurl = "https://api.github.com/repos/%s/%s" % (
+            self.user_name, self.repositoty_name)
+
+    def get_total_contributions(self, weeks):
+        ''' Helper function to calculate total additions
+        and total deletions from a list of weeks of contributions
+        of a contributor
+
+        Parameters
+        -------------
+        weeks : list
+           List of weeks of contributions. Example:
+           [
+            {'w': 1254009600, 'a': 5, 'c': 2, 'd': 9},
```

Comment:
Just say that it parses the list format returned from the Github API. You might also want to make it "private" so that users know that it's not that useful to them.